lib/http/http.c

```diff
@@ -559,8 +560,11 @@ gotheaders(struct http_cookie * H, uint8_t * buf, size_t buflen)
 	 */
 	if ((clen = http_findheader(H->res.headers, H->res.nheaders,
 	    "Content-Length")) != NULL) {
-		/* Parse the value (strtoull skips LWS). */
```

Comment:
Incidentally, can you remember what LWS means here?

Reply:
Linear White Space.